Extensions/Content/Dnn.PersonaBar.Pages/Pages.Web/src/services/pageService.js

```diff
@@ -23,9 +23,12 @@ const PageService = function () {
         return api.post("SavePageDetails", toBackEndPage(request));
     };
 
-    const deletePage = function (page) {
+    const deletePage = function (page, hardDelete) {
         const api = getPagesApi();
-        return api.post("DeletePage", { id: page.tabId });
+        if(typeof hardDelete === "undefined"){
+            hardDelete = false;
+        }
```

Comment:
We could use default parameter values

```js
const deletePage = function (page, hardDelete = false) {
}
```

Reply:
hard delete used in TabController as well, so i keep the same name here.

Reply:
👍